src/OpenTelemetry.Exporter.OneCollector/OneCollectorExporter.cs

```diff
@@ -0,0 +1,78 @@
+// <copyright file="OneCollectorExporter.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Exporter.OneCollector;
+using OpenTelemetry.Internal;
+using OpenTelemetry.Resources;
+
+namespace OpenTelemetry.Exporter;
```

Comment:
Which one do we want `OpenTelemetry.Exporter` or `OpenTelemetry.Exporter.OneCollector`?

Reply:
Good question. Not totally sure! Kind of inconsistent...

* [Zipkin uses OpenTelemetry.Exporter](https://github.com/open-telemetry/opentelemetry-dotnet/blob/d0829fff2f229b67c884526cb4943b99f6c75f29/src/OpenTelemetry.Exporter.Zipkin/ZipkinExporter.cs#L30) as does Jaeger & Otlp
* [Prometheus uses OpenTelemetry.Exporter.Prometheus but isn't public](https://github.com/open-telemetry/opentelemetry-dotnet/blob/d0829fff2f229b67c884526cb4943b99f6c75f29/src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusExporter.cs#LL20C11-L20C44)
* [Geneva uses OpenTelemetry.Exporter.Geneva](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/9c89abf4780fe17508c8cb9d78d716df9e6663ab/src/OpenTelemetry.Exporter.Geneva/GenevaTraceExporter.cs#LL23C11-L23C40)

I don't recall if any of that was intention (@alanwest?). But "OpenTelemetry.Exporter.[Name]" seems reasonable to me so I updated the PR with one exception:

The current extension (`OneCollectorOpenTelemetryLoggerOptionsExtensions.AddOneCollectorExporter`) I left in `OpenTelemetry.Logs`. [Geneva uses Microsoft.Extensions.Logging](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/9c89abf4780fe17508c8cb9d78d716df9e6663ab/src/OpenTelemetry.Exporter.Geneva/GenevaLoggingExtensions.cs#L23) but that seems kind of weird to me. My practice generally is to put the extensions in the namespace of the thing you are extending.

Reply:
@CodeBlanch the updates you made make a lot of sense to me 👍